### PR TITLE
Minor changes to bookmark modal info section

### PR DESCRIPTION
**Pull request by agrogers**
Screenshots show it all. 

image

image

### CODE REVIEW

**Comment by agrogers** [comment]
The are .gitignore file ended up in this pull request. I don't know how to get rid of that. And maybe I have not done other things that should be done. Haven't made many pull requests before.

File: app/bibleview-js/src/lang/default.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 openMyNotes: Open in My Notes
-openStudyPad: Open in Study Pad (%s)
+openStudyPad: "Open '%s' study pad"
```
Comment (tuomas2) [inline app/bibleview-js/src/lang/default.yaml:2]: These changes I think we could do in Transifex (en_GB)

File: .gitignore [hunk 2]
```diff
@@ -16,3 +16,17 @@
...
+app/gradlew.bat
+app/gradlew
+app/gradle/wrapper/gradle-wrapper.properties
+app/gradle/wrapper/gradle-wrapper.jar
+app/build.gradle
+app/bibleview-js/package.json
+app/bibleview-js/package-lock.json
+app/.idea/vcs.xml
+app/.idea/runConfigurations.xml
+app/.idea/modules.xml
```
Comment (tuomas2) [inline .gitignore]: These changes should be reverted. You can simply remove these lines from .gitignore and commit.

**Comment by tuomas2** [comment]
Thanks & congrats for your first PR :-)

File: .gitignore (hunk 2 above)
Comment (agrogers) [reply .gitignore]: Ok, I deleted them from my .gitignore file and then committed them. Is that all?

File: app/bibleview-js/src/lang/default.yaml (hunk 1 above)
Comment (agrogers) [reply app/bibleview-js/src/lang/default.yaml:2]: Done

**Comment by tuomas2** [comment]
Merged manually

**Pull request closed without merging**